src/main/java/io/confluent/kafkarest/KafkaRestMain.java

```diff
@@ -50,7 +52,10 @@ public boolean verify(
    */
   public static void main(String[] args) throws IOException {
     try {
+      /*TODO discuss with team and look at options to do something like Apache CLI here. Handle
```

Comment:
I think you can get rid of this comment, after making changes suggested in KafkaRestApplication.